frontend/lib/ui/Modal/Modal.tsx

```diff
@@ -7,17 +7,23 @@ import { IconButton } from "@kiesraad/ui";
 import cls from "./Modal.module.css";
 
 export interface ModalProps {
+  id: string;
   onClose?: () => void;
   children: React.ReactNode;
 }
 
-export function Modal({ onClose, children }: ModalProps): React.ReactNode {
+export function Modal({ id, onClose, children }: ModalProps): React.ReactNode {
   const modalRoot = document.body;
 
+  React.useEffect(() => {
+    document.getElementById(id)?.focus();
+  }, [id]);
+
   return createPortal(
     <div className={cls.modal} role="dialog">
       <div className={cls["modal-container"]}>
         {onClose && (
+          // TODO: How to make sure thus button can focus with keyboard?
```

Comment:
@lkleuver Do you know how I can make sure this close button can focus with the keyboard?